src/main/resources/messages.properties

```diff
@@ -6,7 +6,8 @@ invalid.max-size=${validatedValue} must be less than {max} symbols
 invalid.email=Please enter valid email address (Ex: username@example.com)
 invalid.username=Please enter valid login, do not use space
 invalid.past-or-present.date=${validatedValue} date couldn't be in the future
-invalid.password=Your password must be between 8 and 30 characters at must contain at least: 1 uppercase Latin character (A-Z), 1 lowercase Latin character (a-z), 1 digit (0-9)
+invalid.password=Your password must contain at least: 1 uppercase Latin character (A-Z), 1 lowercase Latin character (a-z), 1 digit (0-9)
+invalid.password.size=Your password must be between 8 and 30 characters long
```

Comment:
Could you please add the same message changes to test/resources/messages.properties

Reply:
Thank you, very attentive remark =) Done.